### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by refs**

### CODE REVIEW

**Review by C0rby — APPROVED** [review]
LGTM :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Changelog for unreleased
+# Changelog for 2.0.0
 
-The following sections list the changes for unreleased.
+The following sections list the changes for 2.0.0.
```
Comment (micbar) [inline CHANGELOG.md:4]: This file will be auto generated by calens, no need to. bump version here.

**Review by micbar — COMMENTED** [review]
See comment

File: CHANGELOG.md (hunk 1 above)
Comment (refs) [reply CHANGELOG.md:4]: just me not trusting the `machines`. I always run the changelog locally. But won't do it next time :)

**Pull request merged**